### PR TITLE
Document XRView.isFirstPersonObserver

### PR DESCRIPTION
**Pull request by Elchi3**
Spec https://immersive-web.github.io/webxr-ar-module/#dom-xrview-isfirstpersonobserver

### CODE REVIEW

File: files/en-us/web/api/xrview/isfirstpersonobserver/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- WebXR API
+- WebXR Device API
+- XR
+- XRView
+- Augmented Reality
+browser-compat: api.XRView.isFirstPersonObserver
+---
+<p>{{APIRef("WebXR Device API")}}</p>
+
+<p>The {{domxref("XRView")}} interface's read-only <code><strong>isFirstPersonObserver</strong></code> property is a boolean indicating if the <code>XRView</code> is a first person observer view.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/xrview/isfirstpersonobserver/index.html:20]:
```suggestion
<p>The {{domxref("XRView")}} interface's read-only <code><strong>isFirstPersonObserver</strong></code> property is a boolean indicating if the <code>XRView</code> is a first-person observer view.</p>
```

File: files/en-us/web/api/xrview/isfirstpersonobserver/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<p>To create video recordings of AR device cameras you can't simply use one of the rendered eye as there often will be a physical offset. Some devices expose a secondary view, the first-person observer view, which has an <code>eye</code> of <code>none</code>.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/xrview/isfirstpersonobserver/index.html:22]:
```suggestion
<p>To create video recordings of AR device cameras, you can't simply use one of the rendered eyes, as there often will be a physical offset. Some devices expose a secondary view, the first-person observer view, which has an <code>eye</code> of <code>none</code>.</p>
```

File: files/en-us/web/api/xrview/isfirstpersonobserver/index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+<p>To receive a first-person observer view, you need to enable the "secondary-views" feature descriptor explicitly (typically as an optional feature). See {{domxref("XRSystem.requestSession()")}} for details.</p>
+
+<p>The <code>isFirstPersonObserver</code> property then allows you to check which secondary view is a first person observer view.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/xrview/isfirstpersonobserver/index.html:26]:
```suggestion
<p>The <code>isFirstPersonObserver</code> property then allows you to check which secondary view is a first-person observer view.</p>
```

File: files/en-us/web/api/xrview/isfirstpersonobserver/index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+<h2 id="Examples">Examples</h2>
+
+<h3>Checking for first person observer views</h3>
```
Comment (sideshowbarker) [inline files/en-us/web/api/xrview/isfirstpersonobserver/index.html:30]:
```suggestion
<h3>Checking for first-person observer views</h3>
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by teoli2003** [comment]
I wonder if we should define somewhere (not here), _first-person observer views_ and _secondary views_. The meaning is not obvious. Maybe 2 glossary entries (that we can link to, when first used on a page)?

**Comment by Elchi3** [comment]
hm yes, either glossary pages or a guide that explains the eyes and views altogether? I'm opening an issue to collect missing WebXR guide material, so I can first continue with completing the reference pages.

**Pull request merged**